disnake/member.py

```diff
@@ -321,6 +322,9 @@ def __init__(self, *, data: MemberWithUserPayload, guild: Guild, state: Connecti
         self.nick: Optional[str] = data.get("nick", None)
         self.pending: bool = data.get("pending", False)
         self._avatar: Optional[str] = data.get("avatar")
+        self._communication_disabled_until: Optional[datetime.datetime] = utils.parse_time(
+            data.get("communication_disabled_until", None)
+        )
```

Comment:
Maybe split this over two lines rather then three? Kinda hate what black did here.

Reply:
What do you mean? I can't do much if black changes it.

Reply:
```suggestion
        ... = utils.parse_time(data.get("communication_disabled_until", None))
        self._communication_disabled_until: Optional[datetime.datetime] = ...
```